database-plugins/pom.xml

```diff
@@ -58,6 +63,12 @@
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
     </dependency>
+    <!-- https://mvnrepository.com/artifact/org.apache.ibatis/ibatis-sqlmap -->
+    <dependency>
+      <groupId>org.apache.ibatis</groupId>
+      <artifactId>ibatis-sqlmap</artifactId>
+      <version>2.3.0</version>
```

Comment:
I think it's better to not include the version here and have a copy of the dependency in the top level pom for Hydrator Plugins with the version number.